crates/sui-graphql-rpc/src/context_data/db_data_provider.rs

```diff
@@ -13,14 +13,15 @@ use crate::{
         committee_member::CommitteeMember,
         date_time::DateTime,
         digest::Digest,
+        dynamic_field::DynamicField,
         end_of_epoch_data::EndOfEpochData,
         epoch::Epoch,
         event::{Event, EventFilter},
         gas::{GasCostSummary, GasInput},
         move_module::MoveModuleId,
         move_object::MoveObject,
         move_package::MovePackage,
-        move_type::MoveType,
+        move_type::MoveType,        
```

Comment:
nit: spurious whitespace -- your life will be easier if you instruct your editor to get rid of these for you automatically on save.